scripts/web-bench.sh

```
@@ -0,0 +1,149 @@
+  # path below intentionally leaves bodies compressed; discovery must not).
+  curl -ks --compressed -A 'carwatch-web-bench/1' "$base/" 2>/dev/null \
+    | grep -oE '/assets/[^"]+\.(js|css)' | head -1
+}
+
+bench_path() {
+  local label="$1" base="$2" path="$3" kind="$4"
+  local url="${base%/}$path"
+  local ttfbs=() totals=() size="" code="" enc_cdn=""
+  enc_cdn="$(probe_headers "$url" || true)"
+  for _ in $(seq 1 "$SAMPLES"); do
+    read -r dns conn tls ttfb total sz hc < <(curl "${CURL_OPTS[@]}" -o /dev/null -w "$TIMEFMT" "$url" 2>/dev/null || echo "0 0 0 0 0 0 000")
```

Shellcheck (0.11.0)

[warning] dns appears unused. Verify use (or export if used externally).

(SC2034)